File: 코딩 기초 트레이닝/콜라츠 수열 만들기-p181919/콜라츠 수열 만들기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181919

def solution(n):
    answer = [n]
    current = n

    while current > 1:
        if current % 2 == 0:
            current = current / 2
        else:
            current = current * 3 + 1
        answer.append(current)

    return answer

File: 코딩 기초 트레이닝/콜라츠 수열 만들기-p181919/콜라츠 수열 만들기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181919

function solution(n) {
    var answer = [n];

    let current = n;

    while(current > 1) {
        if(current % 2 === 0) {
            // 짝수
            current = current / 2;
        } else {
            // 홀수
            current = current * 3 + 1;
        }
        answer.push(current);
    }
    return answer;
}
